src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -31,6 +31,8 @@ import eks from './images/eks.png'
 
 The easiest way to install the Kubernetes integration is to use our automated installer to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus OpenMetrics](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes), and [New Relic log monitoring](/docs/logs).
 
+Looking to install our New Relic One integration with Pixie for fine-grained telemetry data? See our [Auto-telemetry with Pixie install instructions](/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie/#install-auto-telemetry-with-pixie), to get deeper insight into your Kubernetes clusters and workloads with just one install command. No language agents required.
```

Comment:
I'd remove the comma after the link to the install instructions. I'd also link workloads to its docs.

Reply:
good catch. Thanks.